docs/topics/debug-flow-with-idea.md

```diff
@@ -102,6 +102,13 @@ Create a Kotlin [flow](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-cor
 
     ![Debug the coroutine](flow-debug-2.png)
 
+> You can disable a "was optimised out" feature via the `-Xdebug` compiler option.
+> The "was optimised out" feature optimizes variables when you use suspend functions.
+> However, debugging a code with optimized variables is hard because you don't see their values.
```

Comment:
> debugging a code

Maybe it's just me, but this doesn't sound correct. Maybe "a program", "a piece of code," or just "code" without an article? "A code", a countable noun, is, like, a cipher, no?